Library/Homebrew/cmd/audit.rb

```diff
@@ -659,6 +659,25 @@ def audit_line(line, lineno)
     end
   end
 
+  def audit_prefix_has_contents
+    return unless formula.prefix.directory?
+
+    unless Dir.glob("#{formula.prefix}/**/*").any? do |f|
+      if File.directory?(f)
+        false
+      else
+        f = File.basename(f)
+        !(Metafiles.copy?(f) || %w[.DS_Store INSTALL_RECEIPT.json].include?(f))
+      end
```

Comment:
Is my code so hard to understand? :cry:  I use `any?` to return at the first sign to find installation file. 

And what is a installation file? First not directory (first branch). Second not meta file or black listed file(second branch).